src/main/java/appeng/client/render/model/UVLModelLoader.java

```diff
@@ -387,7 +389,7 @@ public void setQuadTint( int tint )
 
 	class UVLMarker
 	{
-		boolean uvlMarker = false;
+		boolean ae2_uvl_marker = false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "ae2_uvl_marker" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00116)